content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -55,8 +55,8 @@ to use this feature with Kubernetes stateless pods:
 * CRI-O: version 1.25 (and later) supports user namespaces for containers.
 
 Please note that containerd v1.7 supports user namespaces for containers,
-compatible with Kubernetes {{< skew currentPatchVersion >}}. It should not be used
-with Kubernetes 1.27 (and later).
+compatible with Kubernetes 1.25 and 1.26. It should not be used with Kubernetes
+1.27 (and later).
 
 Support for this in [cri-dockerd is not planned][CRI-dockerd-issue] yet.
```

Comment:
Aside: we could tweak this wording to better match our style guide.

Reply:
@sftim any concrete suggestions? What is not matching?

Reply:
We [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future). So let's say what's in cri-dockerd now, and not mention plans / expectations.

Reply:
It's valuable information _but_ we (Kubernetes project) do not have the capacity to maintain compatibility information about third party projects.

We can include the link, rewording it to be more [timeless](https://developers.google.com/style/timeless-documentation) - and hence easier to maintain.

If there's an exception to make, because the resources are there and backed by a commitment (eg by SIG Node), we need to justify that exception. We do make these exceptions from time to time and this could be a reasonable place to add another.

Reply:
I wrote it in a timeless way now. Let me know what you think.

Reply:
Please re-open if I'm missing something